mlrun/db/httpdb.py

```diff
@@ -78,6 +80,20 @@ def bool2str(val):
     return "yes" if val else "no"
 
 
+def require_iguazio_v4(function):
+    @functools.wraps(function)
+    def wrapper(*args, **kwargs):
+        authentication_mode = mlrun.mlconf.httpdb.authentication.mode
+        if authentication_mode != AuthenticationMode.IGUAZIO_V4:
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                f"This method is only supported when mlrun.authentication.mode='iguazio-v4', "
+                f"but current mode is '{authentication_mode}'."
```

Comment:
```suggestion
                "This method is only supported in an Iguazio V4 system."
```

Say you're running in OpenSource (CE) - setting this mode won't really enable this methods without the rest of the Iguazio services.